pex/venv/installer.py

```diff
@@ -829,7 +829,10 @@ def sys_executable_paths():
                 if pex_interpreter
                 else pex_overrides.get("PEX_MODULE", {entry_point!r} or PEX_INTERPRETER_ENTRYPOINT)
             )
-
+                  
+            for name, value in {inject_env!r}:
```

Comment:
Actually, on closer review, shouldn't this move all the way up to just before line 820? It looks like `PEX_SCRIPT=<script>` / PEXes built with `-c <script>` still don't get env vars exposed in this venv case unlike the `pex.pex.PEX` case (zipapp), which you fixed properly above.

Reply:
@erikwilliamson I'm going to get out the 2.1.151 release with `--exclude` support today before a longish break. If you want time to work on this last issue with the PR; you've got it. If you'd rather it get in this afternoon's release, I can collaborate with an edit if you wish.

Reply:
ok - I'll tackle this in about 45 minutes or so to see if we can get it done today!